### ISSUE TITLE
Table API missing

### ISSUE THREAD

**Issue by Itay2805** [mellinoe/ImGui.NET#274]
It seems that the ImGui table api (`BeginTable` and so on) is missing...

**Comment by JulianVallee** [comment]
You could remove the NuGet package, clone this repository (or better yet, make it a submodule of your project) and reference it in your own project until NuGet is updated.

**Comment by Itay2805** [comment]
yeah thats what i did in the end... its just a shame that nuget is not updated properly...

**Comment by JulianVallee** [comment]
Looks like there are efforts being made to automate this (see https://github.com/mellinoe/ImGui.NET-nativebuild/pull/5), so it should just be a matter of time until NuGet will be kept in sync.
